engine/runtime-integration-tests/src/test/scala/org/enso/interpreter/test/instrument/RuntimeServerTest.scala

```diff
@@ -1307,104 +1307,6 @@ class RuntimeServerTest
     )
   }
 
-  it should "send method pointer updates of partially applied autoscope constructors" in {
```

Comment:
Since ab0724c this test yields execution error and I don't see a way (and need) to fix it. There is no such thing as **partially applied autoscope constructor** anymore.

Reply:
You can check that a partially applied autoscope constructor produces an error instead. To verify the current behavior

Reply:
Thanks for providing [it](https://github.com/enso-org/enso/pull/10187/commits/48af079d0d3723a22f99ad9c46f7acdfb9075181).